src/app/chef/[id]/link/page.tsx

```diff
@@ -0,0 +1,9 @@
+import React from "react";
+
+import OuterLink from "@/src/components/outer-link";
+
+const page = () => {
+  return <OuterLink siteName={"ツイッター"} siteUrl={"https://twitter.com"} />;
```

Comment:
後の実装のことを考えて、フラグメントで囲っておいてほしいです！

```suggestion
  return (
    <>
      <OuterLink siteName={"ツイッター"} siteUrl={"https://twitter.com"} />
    </>
  );```